tests/test_octodns_provider_dnsimple.py

```diff
@@ -38,6 +38,10 @@ class TestDnsimpleProvider(TestCase):
             break
 
     def test_populate(self):
+
+        # Sandbox
+        provider = DnsimpleProvider('test', 'token', 42, 'true')
```

Comment:
Please throw in a minimal check here, e.g. 

```suggestion
        provider = DnsimpleProvider('test', 'token', 42, 'true')
        self.assertTrue('sandbox' in provider._client.base)
```

Reply:
yep, wasn't sure how to do that. thx.